src/components/AmountEditable.tsx

```diff
@@ -29,6 +29,8 @@ function methodToIcon(method: MethodChoice["method"]) {
         return "lightning";
     } else if (method === "onchain") {
         return "chain";
+    } else if (method === "fedimint") {
+        return "community";
```

Comment:
you have a comment here saying to add fedimint, you can remove that now.

also would be better as a switch than if statements